assets/src/css/admin/forms.scss

```diff
@@ -17,52 +44,52 @@ Forms CPT
 }
 
 //---------------------------------------------------
-// Field One Offs
+//  Time, Money, Percentage Fields
 //---------------------------------------------------
-.give-time-field {
-	width: 65px;
-}
+.give_options_panel {
+	.give-time-field {
+		width: 65px;
+	}
 
-.give-money-field {
-	width: 75px;
-	margin-right: 0;
-	margin-left: 0;
-}
+	.give-money-field {
+		width: 75px;
+		margin-right: 0;
+		margin-left: 0;
+	}
 
-.give-money-symbol {
-	border: 1px solid #DDD;
-	background: #FCFCFC;
-	margin: 0;
-	font-size: 14px;
-	padding: 5px 8px;
-}
+	.give-money-symbol {
+		border: 1px solid #7e8993;
+		background: #FCFCFC;
+		margin: 0;
+		font-size: 14px;
+		padding: 8px 10px 9px;
+	}
 
-.give-money-symbol-before {
-	border-right: 0;
-	margin-right: -4px;
-}
+	.give-money-symbol-before {
+		border-right: 0;
+		margin-right: -5px;
+		border-radius: 4px 0 0 4px;
 
-.give-money-symbol-after {
-	//border-left: 0;
-	margin-left: -5px;
-}
+		+ .give-money-field {
```

Comment:
@DevinWalker I can see that money field currency icon does not have the same position as input field.

![image](https://user-images.githubusercontent.com/1784821/69620108-d1262980-1062-11ea-9062-b5e699ce0a32.png)